### PR TITLE
Python 3 and Thumbor 7 support, No vows in tests

### PR DESCRIPTION
**Pull request by mbarouski**
Changes to support python 3 and thumbor 7.

Removed pyvows and similar stuff in favor of pytest.

Hope it resolves [this issue](https://github.com/thumbor-community/prometheus/issues/4), and [this](https://github.com/thumbor-community/prometheus/issues/5).

### CODE REVIEW

**Comment by heynemann** [comment]
LGTM

**Review by heynemann — APPROVED** [review]

**Comment by mbarouski** [comment]
@heynemann since approval is here, who has rights to merge and release it?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,24 @@
 # Change Log
+
```
Comment (savar) [inline CHANGELOG.md:2]:
```suggestion
```

The http://keepachangelog.com actually has specifically no new line and this file follows it (as it states).

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 This project adheres to [CHANGELOG](http://keepachangelog.com).
 
 ## [Unreleased]
 
-## [0.1.1] - 2916-03-17
-- fixed hard coded label mapping not matching with prefix 'thumbor_'
+## [1.0.0] - 2022-07-01
+
```
Comment (savar) [inline CHANGELOG.md:10]:
```suggestion
```

The http://keepachangelog.com actually has specifically no new line and this file follows it (as it states).

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,15 +1,24 @@
...
+- Python 3 and Thumbor 7 support
+- Removed pyvows in favor of pytest
+
+## [0.1.1] - 2017-03-17
+
```
Comment (savar) [inline CHANGELOG.md:15]:
```suggestion
```

The http://keepachangelog.com actually has specifically no new line and this file follows it (as it states).

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,15 +1,24 @@
...
+- fixed hard coded label mapping not matching with prefix 'thumbor\_'
 
 ## [0.1.0] - 2017-02-23
+
```
Comment (savar) [inline CHANGELOG.md:19]:
```suggestion
```

The http://keepachangelog.com actually has specifically no new line and this file follows it (as it states).

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,15 +1,24 @@
...
 ### Added
+
```
Comment (savar) [inline CHANGELOG.md:21]:
```suggestion
```

The http://keepachangelog.com actually has specifically no new line and this file follows it (as it states).

File: tests/test_metrics.py [hunk 6]
```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Licensed under the MIT license:
+# http://www.opensource.org/licenses/mit-license
+# Copyright (c) 2017 Simon Effenberg <savar@schuldeigen.de>
```
Comment (savar) [inline tests/test_metrics.py:6]:
```suggestion
```

wasn't me ;)

File: tests/test_metrics.py [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+# Copyright (c) 2017 Thumbor Community Extensions
```
Comment (savar) [inline tests/test_metrics.py:7]:
```suggestion
# Copyright (c) 2022 Thumbor Community Extensions
```

**Review by savar — CHANGES_REQUESTED** [review]
Looks so far good to me (except the suggestions which are tiny things). Yet I am not able to test it at the moment. So @heynemann you would be better telling if it all makes sense.

File: CHANGELOG.md (hunk 1 above)
Comment (mbarouski) [reply CHANGELOG.md:2]: Sure, I will drop those lines, my formatter added them.

**Comment by mbarouski** [comment]
@savar thanks for the feedback, I addressed all points :)

**Review by savar — APPROVED** [review]

**Pull request merged**